File: docs/components_page/components/button_group/dropdown.py
import dash_bootstrap_components as dbc

button_group = dbc.ButtonGroup(
    [
        dbc.Button("First"),
        dbc.Button("Second"),
        dbc.DropdownMenu(
            [dbc.DropdownMenuItem("Item 1"), dbc.DropdownMenuItem("Item 2")],
            label="Dropdown",
            group=True,
        ),
    ]
)

File: docs/components_page/components/button_group/dropdown.jl
using DashBootstrapComponents

button_group = dbc_buttongroup([
    dbc_button("First"),
    dbc_button("Second"),
    dbc_dropdownmenu(
        [dbc_dropdownmenuitem("Item 1"), dbc_dropdownmenuitem("Item 2")],
        label = "Dropdown",
        group = true,
    ),
]);
